docs/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>unnecessary composition **after we eval()'ed it**.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>What do you mean?</p>

Reply:
<p>I also do not understand the suggestion :)</p>

Reply:
Ok! I added this to the remark. Is it better now?